en-US/GetStarted.htm

```diff
@@ -25,10 +26,10 @@
                     <img srcset="{{site.baseurl}}/Resources/images/get-started/BuildPrototype_491x276.jpg" src="{{site.baseurl}}/Resources/images/get-started/BuildPrototype_491x276.jpg" alt="Build prototype image">
                 </picture>
                 <div>
-                    <h3 class="c-heading">Build your Prototype</h3>
-                    <p class="c-paragraph">If you're new to Windows 10 IoT Core, we'll guide you on how to set up your device, write your first app, and connect to Azure.</p>
+                    <br><h2>Build your prototype</h2>
```

Comment:
Why remove the css class ?

Reply:
Rushmi requested that the links as they were currently (with links being underlined twice - see current page for example) was a jarring experience. This new edit fixes that.